src/Entities/Remitter/RemitterAccountIdentifierBuilder.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TrueLayer\Entities\Remitter;
+
+use TrueLayer\Entities\EntityBuilder;
+use TrueLayer\Exceptions\InvalidArgumentException;
+use TrueLayer\Interfaces\Remitter\BbanRemitterInterface;
+use TrueLayer\Interfaces\Remitter\IbanRemitterInterface;
+use TrueLayer\Interfaces\Remitter\NrbRemitterInterface;
+use TrueLayer\Interfaces\Remitter\RemitterAccountIdentifierBuilderInterface;
+use TrueLayer\Interfaces\Remitter\ScanRemitterInterface;
+
+class RemitterAccountIdentifierBuilder extends EntityBuilder implements RemitterAccountIdentifierBuilderInterface
```

Comment:
can we use the existing AccountIdentifier\AccountIdentifierBuilder and the respective entities/interfaces? I think they should be identical.